platform/android/export/export.cpp

```diff
@@ -744,6 +744,21 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return OK;
 	}
 
+	// Writes string p_data to a file at p_path, creating directories as necessary.
```

Comment:
This won't be necessary once [this PR](https://github.com/godotengine/godot/pull/39864) is merged.